### PR TITLE
[CHK-3086] configure logging

### PR DESCRIPTION
**Pull request by a-mannia**
#### List of Changes

- Adds the application loggin configuration

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/main/kotlin/it/pagopa/helpdeskcommands/controllers/CommandsController.kt [hunk 1]
```diff
@@ -13,10 +13,19 @@ class CommandsController() : CommandsApi {
     private val logger = LoggerFactory.getLogger(this.javaClass)
 
     override fun refundOperation(
+        xUserId: String?,
```
Comment (pietro-tota) [inline src/main/kotlin/it/pagopa/helpdeskcommands/controllers/CommandsController.kt:16]:
```suggestion
        xUserId: String,
```

Comment (pietro-tota) [reply src/main/kotlin/it/pagopa/helpdeskcommands/controllers/CommandsController.kt:16]: make also the below parameter as non-nullable too

Comment (a-mannia) [reply src/main/kotlin/it/pagopa/helpdeskcommands/controllers/CommandsController.kt]: Done here: dd050b01

**Comment by infantesimone** [comment]
@a-mannia please resolve conflicts

**Review by infantesimone — APPROVED** [review]

**Pull request merged**